runelite-client/src/main/java/net/runelite/client/plugins/regenmeter/RegenMeterPlugin.java

```diff
@@ -41,17 +42,20 @@
 import net.runelite.client.config.ConfigManager;
 import net.runelite.client.plugins.Plugin;
 import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.plugins.runenergy.RunEnergyPlugin;
```

Comment:
You should definitely not reference other plugins like this

Reply:
changed to be like other utils